client/modules/IDE/components/PreviewFrame.jsx

```diff
@@ -207,6 +207,12 @@ class PreviewFrame extends React.Component {
     accessiblelib.setAttribute('src', 'https://cdn.rawgit.com/MathuraMG/p5-accessibility/9cb5bd0b/dist/p5-accessibility.js');
     const accessibleOutputs = sketchDoc.createElement('section');
```

Comment:
should these accessible outputs be created in the library itself? this is necessary for the library to work, right?

Reply:
The reason we don't create the sections in the library is because the section is [currently the trigger](https://github.com/mathuramg/p5-accessibility#usage) to which output the user wants. So in this case we create that section when they select the option in preferences.

@lm-n

Reply:
got it. would it instead make sense to make some boolean for the library instead that enables/disables the accessible output? then, in the future, if this ever needs to change you can change it in the accessibility library rather than here as well.

Reply:
Sorry, I didn't get what you mean by ' boolean for the library'? :)
The reason we are having sections now is so that beyond the editor the user can determine where they want the output in their program.

Reply:
got it! what i mean is maybe there's a script that initializes the library, rather than just appending to an HTML element if it's there. for example, one includes the accessibility library, and then includes a script:
```JS
var accessibility = new p5.Accessibility(<reference to the HTML element you want to append to>);
```
or it could create an HTML element to append to by default. My inspiration for this is [dat.gui](https://github.com/dataarts/dat.gui). my instinct is that it is not the best that including an html element determines whether or not the library is included.

Reply:
Got it!  Yes, we will look more into this. 👍 
Would be okay if we track these refactors in a different issue that we tackle following this?

Reply:
sure! should i merge this in then?

Reply:
sounds good!

Reply:
Here are the issues on the library repo: https://github.com/MathuraMG/p5-accessibility/issues/13 https://github.com/MathuraMG/p5-accessibility/issues/12

Reply:
@catarak Yes! That sounds good!

Reply:
great, will do ⭐️